platforms/macos/src/util.rs

```diff
@@ -35,36 +36,37 @@ pub(crate) fn to_ns_range_for_character(pos: &TextPosition) -> NSRange {
 }
 
 pub(crate) fn from_ns_point(view: &NSView, node: &Node, point: NSPoint) -> Point {
-    let window = view.window().unwrap();
-    let point = window.convert_point_from_screen(point);
-    let point = view.convert_point_from_view(point, None);
+    let window = unsafe { view.window() }.unwrap();
+    let point = unsafe { window.convertPointFromScreen(point) };
+    let point = unsafe { view.convertPoint_fromView(point, None) };
     // AccessKit coordinates are in physical (DPI-dependent) pixels, but
     // macOS provides logical (DPI-independent) coordinates here.
-    let factor = view.backing_scale_factor();
+    let factor = unsafe { window.backingScaleFactor() };
     let point = Point::new(
         point.x * factor,
-        if view.is_flipped() {
+        if unsafe { view.isFlipped() } {
             point.y * factor
         } else {
-            let view_bounds = view.bounds();
+            let view_bounds = unsafe { view.bounds() };
             (view_bounds.size.height - point.y) * factor
         },
```

Comment:
FYI: These methods are always safe, and will be marked as such in `icrate` in the future ;)